test/systemtests/cfg.json

```diff
@@ -1 +1 @@
-{"binpath": "/opt/gopath/bin", "iterations": 3, "encap": "vlan", "keyFile": "/home/admin/.ssh/id_rsa", "gateway": "10.1.1.254", "subnet": "10.1.1.0/24", "network": "TestNet", "contiv_cluster_store": "etcd://localhost:2379", "platform": "vagrant", "mgmtInterface": "eth1", "dataInterfaces": "eth2,eth3", "vxlan": "1-10000", "product": "netplugin", "swarm_variable": "", "vlan": "1120-1150", "aci_mode": "off", "scheduler": "docker", "enableDNS": false, "tenant": "TestTenant", "short": false, "hostips": "192.168.2.10,192.168.2.11,192.168.2.12", "hostusernames": "admin,admin,admin", "containers": 3, "contiv_l3": ""}
\ No newline at end of file
+{"scheduler": "k8s", "install_mode": "kubeadm", "swarm_variable": "", "platform": "vagrant", "product": "netplugin", "aci_mode": "off", "short": false, "containers": 3, "iterations": 3, "enableDNS": false, "contiv_l3": "", "contiv_cluster_store": "etcd://192.168.2.10:6666", "keyFile": "/home/admin/.ssh/id_rsa", "binpath": "contiv/bin", "hostips": "192.168.2.10,192.168.2.11,192.168.2.12", "hostusernames": "admin,admin,admin", "dataInterfaces": "eth2", "mgmtInterface": "eth1", "vlan": "1120-1150", "vxlan": "1-10000", "subnet": "10.1.1.0/24", "gateway": "10.1.1.254", "network": "TestNet", "tenant": "TestTenant", "encap": "vlan"}
```

Comment:
the k8s_* vars are only in createcfg.py which generates the cfg.json (which in turn is consumed by system-tests). We should remove adding this cfg.json in our repository as it changes every time a system test is run. I've already added this in gitignoe, but I guess we have to remove from the git first for not tracking this.